### ISSUE TITLE
support openapi 3.0

### ISSUE THREAD

**Issue by zhenhua32** [maximdanilchenko/aiohttp-apispec#46]
Is there any plan support openapi 3.0 ?

**Comment by maximdanilchenko** [comment]
Yes, I have such plans, and it is quite easy to implement (`apispec` has this feature). There will be an extra argument with openapi version.

**Comment by rugleb** [comment]
Any updates?

**Comment by ondrejkajinek** [comment]
#123 Similar PR, up to date with code, also provides correct schema for response body.

**Comment by arcan1s** [comment]
according to [apispec docs](https://apispec.readthedocs.io/en/latest/api_ext.html#apispec.ext.marshmallow.openapi.OpenAPIConverter.schema2parameters) `in: body` parameter is not allowed for openapi 3.0. In fact, stoplight elements (https://elements-demo.stoplight.io/) doesn't render such parameters and is awaiting them in `requestBody` section
